doc/tools/stylelint.md

```diff
@@ -0,0 +1,17 @@
+# stylelint
+
+## install stylelint
+```sh
+  npm install --save-dev stylelint stylelint-config-standard
```

Comment:
Maybe tell the user to go to /front directory, and what about moving this documentation to the front directory also ?